scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/executors/forked/env/ForkedTaskVariablesManager.java

```diff
@@ -133,11 +134,13 @@ public RemoteSpace createDataSpaceNodeClient(TaskContext container, SchedulerNod
     }
 
     public void replaceScriptParameters(Script<?> script, Map<String, String> thirdPartyCredentials,
-            VariablesMap variables, PrintStream errorStream) {
+            VariablesMap variables, Map<String, Serializable> JobMap, PrintStream errorStream) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00117)